lib/reporter.ts

```diff
@@ -103,7 +103,7 @@ class ReportPortalReporter extends Reporter {
 
   onSuiteStart(suite) {
     log.debug(`Start suite ${suite.title} ${suite.uid}`);
-
+    this.specFilePath = suite.file ? suite.file : this.specFilePath;
```

Comment:
@mikesalvia 
From PR description:
>Instead of finding this.specFilePath once we find it every time we start a new suite.

can we acquire `specFilePath ` only from onSuiteStart even? why do we need get it from onRunnerStart then overwrite in onSuiteStart?

Reply:
For unit tests was empty in onSuiteStart so I added back the onRunnerStart.

Reply:
@BorisOsipov Let me know if any changes are required.

Reply:
@mikesalvia I will check this tommorow. From the first sight it seems we should remove it from onRunnerStart and fix unit tests.